Monika After Story/game/script-holidays.rpy

```diff
@@ -2651,8 +2651,15 @@ label monika_aiwfc_song:
     $ disable_esc()
     $ mas_MUMURaiseShield()
 
-    $ play_song(None, 1.0)
+    # always unmute the music channel (or at least attempt to)
+    # TODO: there should probably be handling for sayori name case.
+    if songs.getVolume("music") == 0.0:
+        $ renpy.music.set_volume(1.0, channel="music")
+
+    # save background sound for later
     $ amb_vol = songs.getVolume("backsound")
```

Comment:
We need to get the exact volume of the amb mixer so we can revert back to it later.
```suggestion
    $ amb_vol = songs.getUserVolume("backsound")
```

Reply:
as mentioned in the pr, there is a difference between channel volume and user volume. using set_volume sets the channel volume, which is not controlled by the user. 

the code here sets channel volume so getting the user volume and setting the channel volume to it could change the volume in a way the user cannot revert back.